docs/details/develop-checks.md

```diff
@@ -1,266 +1,313 @@
-    - _PERCONA_TEST_CHECKS_DISABLE_START_DELAY=true_ to disable the default check execution start delay, which is currently set to 1 minute, so that checks are run immediately upon system start
+    - _PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml_ to use checks from the local files instead of downloading them from the SaaS.
+    - _PERCONA_TEST_CHECKS_DISABLE_START_DELAY=true_ to disable the default check execution start delay. This is currently set to one minute, so that checks run upon system start.
     - _PERCONA_TEST_CHECKS_RESEND_INTERVAL=2s_ to define the frequency for sending the SA-based alerts to Alertmanager.
-
+ 
     ```
     docker run -p 80:80 -p 443:443 --name pmm-server \
-
     -e PMM_DEBUG=1 \
     -e PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml \
-    -e PERCONA_TEST_CHECKS_DISABLE_START_DELAY=true
+    -e PERCONA_TEST_CHECKS_DISABLE_START_DELAY=true \
     -e PERCONA_TEST_CHECKS_RESEND_INTERVAL=2s \
     perconalab/pmm-server:dev-latest
     ```
-
-3.  Log in to Grafana (admin/admin) and enable STT in the settings: http://127.0.0.1/graph/d/pmm-settings/pmm-settings
-
-    ![!](../_images/Grafana.png)
-
-4.  Create _/srv/custom-checks.yml_ inside a Docker container with the content from the Security Advisor (Security Threat Tool) section above.
-
-5.  STT checks will run with a time interval defined via UI. You can see the result of running the advisor on the home dashboard:
-
+ 
+3.  Log in to Grafana with credentials **admin/admin**.
+ 
+4. Go to **Configuration > Settings > Advanced Settings** and enable **Advisors**. For PMM 2.26 and older this option is called **Security Threat Tool**.
+ 
+4.  Create _/srv/custom-checks.yml_ inside a Docker container with the content of your check.
+ 
+5.  The checks will run according to the time interval defined on the UI. You can see the result of running the check on the home dashboard:
+ 
     ![!](../_images/HomeDashboard.png)
-
+ 
 6.  Click on the number of failed checks to open the Failed Checks dashboard:
-
+ 
     ![!](../_images/FailedChecks.png)
-
-7.  Go into Docker container to output the logs of pmm-managed and read STT logs:
-
+ 
+7.  Go into Docker container to output the logs of pmm-managed and read check logs:
+ 
 ```
 # get inside the container
 docker exec -it pmm-server bash
 # print and watch the logs
 supervisorctl tail -f pmm-managed
+ 
 ```
+## Develop security checks for PMM 2.26 and older
+
+### Advisor checks versus security checks
+PMM 2.26 and older included a set of security checks grouped under the **Security Threat Tool** option.
+ 
+Starting with the 2.27 release PMM introduced new checks and grouped them into set of Advisors, according to the functionality and recommendations they provide.
+ 
+To reflect these changes, the old **Security Threat Tool** option in PMM 2.26 in earlier has been renamed to **Advisors** and the checks use a slightly different format.
```

Comment:
For now we have only one format. Also we plan to keep support of V1 event when V2 will be released.